### PR TITLE
solution-1

### PR DESCRIPTION
**Pull request by ritoneet**
Done

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,38 @@
...
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: "Car") -> float:
+        price = (car.comfort_class * (self.clean_power - car.clean_mark)
+                 * (self.average_rating / self.distance_from_city_center))
+        return round(price, 1)
+
+    def wash_single_car(self, car: "Car") -> None:
```
Comment (vsmutok) [inline app/main.py]:
```suggestion
    def wash_single_car(self, car: Car) -> None:
```

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,38 @@
...
+
+    def serve_cars(self, cars: list) -> float:
+        income = 0.0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return round(income, 1)
+
+    def calculate_washing_price(self, car: "Car") -> float:
```
Comment (vsmutok) [inline app/main.py]:
```suggestion
    def calculate_washing_price(self, car: Car) -> float:
```

File: app/main.py [hunk 3]
```diff
@@ -1,8 +1,38 @@
...
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: float, clean_power: int,
+                 average_rating: float, count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
```
Comment (vsmutok) [inline app/main.py]:
```suggestion
    def serve_cars(self, cars: list[Car]) -> float:
```

**Review by vsmutok — CHANGES_REQUESTED** [review]

**Comment by ritoneet** [comment]
I changed it but how `def calculate_washing_price(self, car: "Car")` and `def calculate_washing_price(self, car: Car)` are different?

**Review by vsmutok — APPROVED** [review]
Good Job!

**Pull request closed without merging**